src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/ImageTests/LoadFromBytes.xaml.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Uno.UI.Samples.Controls;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Media.Imaging;
+using Uno.Extensions;
+
+namespace Uno.UI.Samples.UITests.ImageTestsControl
+{
+	[SampleControlInfo("Image", "LoadFromBytes")]
+	public sealed partial class LoadFromBytes
+	{
+		public LoadFromBytes()
+		{
+			InitializeComponent();
+			Loaded += OnLoaded;
+		}
+
+		private async void OnLoaded(object sender, RoutedEventArgs e)
+		{
+			var assembly = typeof(LoadFromBytes).Assembly;
+
+			var query = from name in assembly.GetManifestResourceNames()
+						where name.EndsWith("mslug.png")
+						select assembly.GetManifestResourceStream(name);
+
+			using (var stream = query.First())
+			{
+				MyImage.Source = await CreateImageSource(stream);
+			}
+		}
+
+		private async Task<ImageSource> CreateImageSource(Stream stream)
+		{
+			if (stream == null)
+			{
+				throw new ArgumentNullException(nameof(stream));
+			}
+
+			var image = new BitmapImage();
+
+#if WINDOWS_UWP
+			await image.SetSourceAsync(stream.AsRandomAccessStream()).AsTask();
```

Comment:
I'm not sure if this is the correct way to assign the stream in UWP

Reply:
This is the correct way for now, until we support AsRandomAccessStream correctly